self-hosting/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,173 @@
+version: '3'
+x-postgres:
+  &postgres-common
+  image: "ankane/pgvector:v0.5.1"
+  user: postgres
+  healthcheck:
+    test: "exit 0"
+    interval: 2s
+    timeout: 12s
+    retries: 3
+
+services:
+  postgres:
+    <<: *postgres-common
+    container_name: "omnivore-postgres"
+    ports:
+      - "5432:5432"
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    env_file:
+      - .env
+
+  migrate:
+    build:
+      context: ../../
+      dockerfile: ./packages/db/Dockerfile
+    container_name: "omnivore-migrate"
+    command: '/bin/sh ./packages/db/setup.sh' # Also create a demo user with email: demo@omnivore.app, password: demo_password
+    env_file:
+      - .env
+    depends_on:
+      postgres:
+        condition: service_healthy
+
+  api:
+    build:
+      context: ../../
+      dockerfile: ./packages/api/Dockerfile
+    container_name: "omnivore-api"
+    ports:
+      - "4000:8080"
+    healthcheck:
+      test: ["CMD-SHELL", "nc -z 0.0.0.0 8080 || exit 1"]
+      interval: 15s
+      timeout: 90s
+    env_file:
+      - .env
+    depends_on:
+      migrate:
+        condition: service_completed_successfully
+
+  queue-processor:
+    build:
+      context: ../../
+      dockerfile: ./packages/api/queue-processor/Dockerfile
+    container_name: "omnivore-queue-processor"
+    env_file:
+      - .env
+    depends_on:
+      api:
+        condition: service_started
+
+  web:
+    build:
+      context: ../../
+      dockerfile: ./packages/web/Dockerfile-self
+      args:
+        - APP_ENV=prod
+        - BASE_URL=http://localhost:3000
+        - SERVER_BASE_URL=http://localhost:4000
+        - HIGHLIGHTS_BASE_URL=http://localhost:3000
+    container_name: "omnivore-web"
+    ports:
+      - "3000:8080"
+    env_file:
+      .env
+    depends_on:
+      api:
+        condition: service_healthy
+
+  image-proxy:
+    build:
+      context: ../../imageproxy
+      dockerfile: ./Dockerfile
+    container_name: "omnivore-image-proxy"
+    ports:
+      - "7070:8080"
+    env_file:
+      - .env
+
+  content-fetch:
+    platform: linux/amd64
```

Comment:
Is there any reason to specify platform? This is causing failure in ARM based machines

Reply:
I've tested this on arm64 and can confirm this line is not necessary, content-fetch works fine on aarch64